packages/aila/package.json

```diff
@@ -4,7 +4,6 @@
   "license": "MIT",
   "main": "./index.ts",
   "types": "./index.ts",
-  "type": "module",
```

Comment:
It might be worth pulling in @stefl as I know he had some trouble here getting the imports working across the CLI, tests, storybook, whatever tools we use

So long as the github checks pass I'm satisfied though

Reply:
As long as we don't have to rename everything .mjs and start importing "something.js" then I'm happy too!